src/hmrc-design-patterns/welsh-language-toggle/index.njk

```diff
@@ -20,16 +20,9 @@ layout: twoColumnPage.njk
 
 {{ hmrcFrontendInfo() }}
 
-<p class="govuk-body"> Put the Welsh language toggle  on the right hand side below the: </p>
-<ul class="govuk-list govuk-list--bullet">
-  <li><a class="govuk-link" href="https://design-system.service.gov.uk/components/header/">GOV.UK header</a></li>
-  <li> <a class="govuk-link" href="https://design-system.service.gov.uk/components/phase-banner/">phase banner</a></li>
-  <li> <a class="govuk-link" href="/hmrc-design-patterns/hmrc-banner/">HMRC banner</a> if you are using it</li>
-</ul>
+<p class="govuk-body">The Welsh language toggle sits inside the service navigation component. Follow the GOV.UK guidance about how to position the <a href="https://design-system.service.gov.uk/components/service-navigation/" class="govuk-link">service navigation component</a> on the page.</p>
 
-<p class="govuk-body">The Welsh language toggle must be above the <a class="govuk-link" href="https://design-system.service.gov.uk/components/back-link/">back link</a>.</p>
-
-<p class="govuk-body">Any information entered before switching language should be retained.</p>
+<p class="govuk-body">When a user switches the language, they should stay on the same page of the service. Any information they have already entered should be kept.</p>
```

Comment:
total aside, though maybe useful to clarify if people will be looking at this page - on this point, it's something I've been curious about - do we mean they shouldn't start from beginning of journey and have to re-enter stuff they have submitted - or do we mean that if user has entered text in a field but not submitted the form that the content will be retained on the page

if it's the latter, then this isn't how language select currently works in mdtp services - services would have to persist data somehow in the users browser because changing language requires the page to be rerendered in welsh - or for the content to be sent along with the request to change language and provided to page somehow when its redisplayed